cudaq-1.0.0/LICENSE

```diff
@@ -0,0 +1,121 @@
+NVIDIA SOFTWARE LICENSE
```

Comment:
Any changes to the software license would be a separate PR, and we'll want to keep it with the Apache 2.0 license we already have checked in. Was this added to this PR accidentally?